core/src/test/scala/org/apache/spark/serializer/KryoSerializerSuite.scala

```diff
@@ -62,6 +62,10 @@ class KryoSerializerSuite extends FunSuite with SharedSparkContext {
     val thrown3 = intercept[IllegalArgumentException](new KryoSerializer(conf4).newInstance())
     assert(thrown3.getMessage.contains(kryoBufferProperty))
     assert(!thrown3.getMessage.contains(kryoBufferMaxProperty))
+    val conf5 = conf.clone()
+    conf5.set(kryoBufferProperty, "8m")
```

Comment:
Doesn't this show it currently works though? Is the resolution that there was no problem (at least not in `master`)? More tests are good but this seems like something already well exercised by tests above.